int16 mode
bool enable
float64[] setpoint
float64[] avtr
float64[] setpoint2

